src/Rendering/utilityLayerRenderer.ts

```diff
@@ -10,6 +10,16 @@ import { Vector3 } from '../Maths/math.vector';
 import { Camera } from '../Cameras/camera';
 import { Color3 } from '../Maths/math.color';
 
+declare module "../scene" {
+    export interface Scene {
+        /**
+         * Associated utility layer renderer
+         * @hidden
+         */
+        _utilityLayerRenderer: UtilityLayerRenderer;

```

Comment:
Just a thought - Wouldn't it be possible to pass the utility layer renderer to the XR feature? In the options or as a setter?

I noticed the question regarding a scene-utility layer renderer relation. Though not expected, it is still possible to have more than one.

Will also avoid adding more parameters to the scene, which is very bloated as it is :-)

Reply:
I looked and there is indeed a property `useUtilityLayer` in `WebXRControllerPointerSelection`, and also a static property `UtilityLayerRenderer.DefaultUtilityLayer.utilityLayerScene``, so we can probably use that